plugins/logs/plugin.go

```diff
@@ -831,13 +831,11 @@ func (p *Plugin) loop() {
 func (p *Plugin) doOneShot(ctx context.Context) error {
 	uploaded, err := p.oneShot(ctx)
 
-	// Make a local copy of the plugins's status. This is needed as locking the status for
-	// the status upload duration will block policy evaluation and result in
-	// increased latency for OPA clients
-	p.mtx.Lock()
```

Comment:
This comment is still valid, isn't it?

Reply:
The status update is no longer protected by the same mutex that is used for logging, so I don't think policy evaluation is affected anymore.